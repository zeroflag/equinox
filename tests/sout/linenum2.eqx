: my-abort ( -- )
  "test" #( error 1 0 ) ;

2 3 + dup *

my-abort


