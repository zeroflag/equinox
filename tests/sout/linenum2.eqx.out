  File "tests/sout/linenum2.eqx", line 2 
      001.  : my-abort ( -- )
    =>002.    "test" #( error 1 0 ) ;
      003.  
  File "tests/sout/linenum2.eqx", line 6 
      005.  
